initfiles/bash/etc/init.d/install-init.in

```diff
@@ -182,6 +182,12 @@ homePath=$(cat /etc/passwd | grep -e "^${user}:" | cut -d':' -f6)
 homeBase=$(dirname $homePath)
 echo ""
 
+# Generate environment.xml
+echo "Generate environment.xml"
+${path}/sbin/envgen -env ${path}/etc/HPCCSystems/rpmnew/environment.xml -ip . -roxienodes 1 -thornodes 1 -slavesPerNode 1 -o data4=/var/lib/[NAME]/hpcc-data4/[COMPONENT] -set_xpath_attrib_value Software/EclCCServerProcess/Option @name maxCompileThreads -set_xpath_attrib_value Software/EclCCServerProcess/Option @value 2 -override roxie,@allFilesDynamic,true -override roxie,@laxyOpen,smart -override roxie,@localSlave,true -override roxie,@slaveConfig,simple > /dev/null 2>&1 || exit 1
```

Comment:
@xwang2713 The envgen call is failing because it's failing to open a file at /etc/HPCCSystems/environment.conf.  We need to make sure that path has been laid down before we do the envgen call.